core/src/com/unciv/ui/worldscreen/WorldScreen.kt

```diff
@@ -540,6 +540,7 @@ class WorldScreen(val viewingCiv:CivilizationInfo) : CameraStageBaseScreen() {
     }
 
     private fun showTutorialsOnNextTurn(){
+        if (!UncivGame.Current.settings.showTutorials) return
```

Comment:
Since this function is called on every `update`, it must be quite optimal, so we can save some performance when the tutorials are not needed at all (I assume it is quite frequent scenario).